### ISSUE TITLE
SideNav on Examples page is not keyboard accessible

### ISSUE THREAD

**Issue by majornista** [adobe/coral-spectrum#6]
### Expected Behavior
User should be able to navigate through SideNav items using the Tab key, and SideNav items should be identified a links using assistive technology.

### Actual Behavior
SideNav items are skipped entirely. Using a screen reader the SideNav items are identified as static text.

### Reproduce Scenario (including but not limited to)
https://opensource.adobe.com/coral-spectrum/dist/examples/#accordion

#### Steps to Reproduce
1. Open https://opensource.adobe.com/coral-spectrum/dist/examples/#accordion
2. Tab into the page, moving focus to the Search input.
3. Press Tab to move focus to the SideNav.
4. Notice that focus skips to the Links for selecting the theme in the upper right hand side of the iframe.

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)
All

#### Coral Spectrum version
1.0.0-beta.116
